lib/iris/fileformats/netcdf.py

```diff
@@ -2104,9 +2194,19 @@ def save(cube, filename, netcdf_format='NETCDF4', local_keys=None,
     with Saver(filename, netcdf_format) as sman:
         # Iterate through the cubelist.
         for cube in cubes:
+            if isinstance(packing, list):
```

Comment:
@jswanljung The user might just use a `tuple` rather than a list, and that'll cause some carnage from this point on ... we could be graceful and deal with that.